### PR TITLE
Update v1.0 troubleshooting/monitoring translation

### PR DESCRIPTION
**Pull request by vickyhella**
- Update the Chinese translation for v1.0 troubleshooting/monitoring

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.0/troubleshooting/monitoring.md [hunk 1]
```diff
@@ -69,3 +69,121 @@ rancher-monitoring-grafana-d9c56d79b-cp86w               3/3     Running   0
...
+# kubectl get volume -A
+NAMESPACE         NAME                                       STATE      ROBUSTNESS   SCHEDULED   SIZE          NODE     AGE
+longhorn-system   pvc-1b2fbbe9-14b1-4a65-941a-7d5645a89977   attached   degraded                 5368709120    harv31   43h
+longhorn-system   pvc-7c6dcb61-51a9-4a38-b4c5-acaa11788978   attached   degraded                 53687091200   harv31   43h
+longhorn-system   pvc-b2b2c07c-f7cd-4965-90e6-ac3319597bf7   attached   degraded                 2147483648    harv31   43h
+```
+
+### 缩减 Deployment
+
+要分离`卷`，你需要缩减使用`卷`的 `Deployment`。
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.0/troubleshooting/monitoring.md:116]:
```suggestion
要停止挂载`卷`，你需要先缩减正在使用`卷`的 `Deployment`。
```

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.0/troubleshooting/monitoring.md [hunk 2]
```diff
@@ -69,3 +69,121 @@ rancher-monitoring-grafana-d9c56d79b-cp86w               3/3     Running   0
...
+
+以下示例针对 `rancher-monitoring-grafana` 声明的 PVC。
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.0/troubleshooting/monitoring.md:118]:
```suggestion
以下示例将针对 `rancher-monitoring-grafana` 使用的 PVC。
```

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.0/troubleshooting/monitoring.md [hunk 3]
```diff
@@ -69,3 +69,121 @@ rancher-monitoring-grafana-d9c56d79b-cp86w               3/3     Running   0
...
+```
+# kubectl get deployment -n cattle-monitoring-system
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+rancher-monitoring-grafana              1/1     1            1           43h  // scaled up
+rancher-monitoring-kube-state-metrics   1/1     1            1           43h
+rancher-monitoring-operator             1/1     1            1           43h
+rancher-monitoring-prometheus-adapter   1/1     1            1           43h
+```
+
+`卷`已附加到新的 POD。
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.0/troubleshooting/monitoring.md:184]:
```suggestion
`卷`已附加到新的 Pod。
```

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.0/troubleshooting/monitoring.md [hunk 4]
```diff
@@ -69,3 +69,121 @@ rancher-monitoring-grafana-d9c56d79b-cp86w               3/3     Running   0
...
+
+![](/img/v1.0/troubleshooting/6-after-scale-up.png)
+
+至此，你已将`卷`扩展到新的大小，POD 可以正常使用卷。
```
Comment (guangbochen) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.0/troubleshooting/monitoring.md:188]:
```suggestion
至此，你已将`卷`扩展到新的大小，Pod 可以正常使用卷。
```

**Comment by vickyhella** [comment]
Suggestions applied and commits squashed. Thanks! @guangbochen

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**